.travis.yml

```diff
@@ -1,36 +1,36 @@
 language: python
```

Comment:
I don't know how this got formatted, and literally copying the file back verbatim from `main` doesn't seem to change it